translations/zh-CN.json

```diff
@@ -1133,7 +1133,7 @@
 	"loading profile": "加载配置文件",
 	"max time to wait per Kubernetes core services to be healthy.": "每个 Kubernetes 核心服务保持健康所需的最长时间。",
 	"max time to wait per Kubernetes or host to be healthy.": "Kubernetes 或主机正常运行前的最大等待时间。",
-	"minikube addons list --output OUTPUT. json, list": "",
+	"minikube addons list --output OUTPUT. json, list": "minikube addons list --output OUTPUT。输出结果为json列表",
```

Comment:
I believe `json, list` should be left as is as those are two options that can be used.